content/_guides/managing-flows.md

```diff
@@ -0,0 +1,147 @@
+| `id`                             | yes          | Flow ID |
+| `attributes.name`                             | no         | Flow name |
+| `attributes.type`                             | no          | Flow type. Allowed values: `ordinary`, `long_running `|
+| `attributes.graph`                             | no          |  	Flow graph representing component connections |
+| `attributes.cron`                             | no          | CRON expression representing Flow timing |
+
+
+## Copy Flow, Switch Flow Type
+
+To copy a Flow within the same Workspace, navigate to see your Flows. Then click on the corresponding icon to open the settings menu of the selected Flow:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_7.png)  
+
+Alternatively, you can navigate to the Flow itself by clicking on its name:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_3.png)
+
+Then switch to *Implement* tab **(1)** and click the command **(2)** to copy it to clipboard, and [create a copy of the Flow via the API](/managing-workspaces):
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_8.png)  
+
+As a result, you get a copy of your Flow:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_9.png)  
+
+To switch between real-time and ordinary types, navigate to see your Flows. Then click on the corresponding icon to open the settings menu of the selected Flow:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_10.png)
+
+Switching back works the same way. Alternatively, you can navigate to the Flow itself by clicking on its name, switch to *Settings* tab **(1)**, scroll down and toggle the corresponding switch **(2)**:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_11.png)
+
+Note, that you can only switch Flow type if there's at least one published Flow version, and it is not running.
+
+## Additional Actions
+
+You can also subscribe to errors, and schedule your Flow via [CRON expressions](https://en.wikipedia.org/wiki/Cron#CRON_expression).
+
+1\. Subscribe to errors is a feature that orders the platform to send you emails in case of any errors in your running Flow. However, it will not just mindlessly spam you with messages if an error repeats. With some errors, messaging interval is one email per hour, while with others its one per 24 hours. This interval is hardcoded and cannot be configured.
+
+To subscribe to errors, use the settings menu **(1)** of the corresponding Flow and click *Subscribe to Errors* **(2)**:
+
+![](/assets/img/tenant-management-guide/managing-flows/Screenshot_12.png)
+
+Unsubscribing works the same way.
+
+2\. To schedule the Flow via CRON expressions, you need to be in drafting mode. You can turn it on by [editing the Flow](#start-stop-edit-delete-via-the-ui). While editing the Flow, switch to *Settings* tab **(1)**, use CRON expression to schedule the Flow, and click *Save*:
```

Comment:
This details about `CRON` are not enough. We need to mention that:
* by default a flow runs every 3 mins because the default cron expression is `*/3 * * * *`
* but you can change this period by specifying your own CRON expression if this default does not fit your needs
* giving few examples and explaining them a bit. We don't need to write a `CRON` tutorial here but some brief intro for beginners would be great
* explaining that when you typed the correct `CRON` expression, you can see the approximate scheduling plan on the right hand side in the `Next Occurrences` section